src/components/AllMovies.vue

```diff
@@ -1,48 +1,147 @@
-                    
+                type="search"
+                class="input__search"
+                >
+            </CustomInput>      
             <SelectInput
                 v-model="selectedCategory"
                 placeholder="All categories"
                 label="category"
                 :options="categoryOptions">
             </SelectInput>
         </div>
+        <section class="movie__wrapper">
+            <MovieCard2 idName="movieCard" class="movie-card"
+                v-for="movie in filteredMovies"
+                :key='"m" + movie.id'
+                :movie="movie">
+
+                <template v-slot:image>
+                    <img :src="movie.poster_url" class="movie-card__img">
+                </template>
+
+                <template v-slot:header >
+                    <h1 class="movie-card__header"> {{movie.title}}</h1>
+                </template>
+            
+                <template v-slot:genre>
+                    <BaseTag :text="movie.genre.name" theme="tag--red"></BaseTag>
+                </template>
+
+                <template v-slot:length>
+                    <BaseTag :text="formatLength(movie.length)"></BaseTag>
+                </template>
+            </MovieCard2>
+        </section>
+
     </section>
 </template>
 
 <script>
+import axios from "axios";
 import BiggestHeader from '@/components/BiggestHeader.vue';
 import CustomInput from '@/components/CustomInput.vue';
 import SelectInput from "@/components/SelectInput.vue";
+import MovieCard2 from '@/components/MovieCard2.vue';
+import BaseTag from '@/components/BaseTag.vue';
 
 export default {
     components: {
         BiggestHeader,
         CustomInput,
-        SelectInput
+        SelectInput,
+        MovieCard2,
+        BaseTag
     },
     data() {
-        return {
+        return { 
+            movieCard: [],
+            MOVIES_URL: 'http://localhost:3000/movies',
             search: '',
             selectedCategory: '',
             categoryOptions: [
-                { text: 'Action', value: 'B' },
-                { text: 'Comedy', value: 'C' },
-                { text: 'Family', value: 'D' },
-                { text: 'Fantasy', value: 'E' },
-                { text: 'Romance', value: 'F' },
-                { text: 'Sci-fi', value: 'G' },
+                { text: 'Action', value: 'action' },
+                { text: 'Comedy', value: 'comedy' },
+                { text: 'Family', value: 'family' },
+                { text: 'Fantasy', value: 'fantasy' },
+                { text: 'Romance', value: 'romance' },
+                { text: 'Sci-fi', value: 'sci-fi' },
             ]
         }
+    },
+    computed: {
+        filteredMovies() {
+            let filteredMovie = this.movieCard.filter(movie => movie.title.toLowerCase().includes(this.search.toLowerCase()))
+            return filteredMovie.filter(movie => movie.genre.name.toLowerCase().includes(this.selectedCategory.toLowerCase()))
```

Comment:
I propose to refactor this line a little bit :) 
`return filteredMovie.filter(movie => movie.genre.name.toLowerCase().includes(this.selectedCategory.toLowerCase()))`

in this way
```
const movies = filteredMovie.filter(movie => movie.genre.name.toLowerCase().includes(this.selectedCategory.toLowerCase()));
return movies;
```
What do you think about it?

Reply:
Yes! This is a lot nicer :)